project_task_description_template/models/project_task.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
+
+from odoo import api, fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+
+    task_template_id = fields.Many2one(comodel_name="project.task.template")
+    template_visible = fields.Boolean(compute="_compute_template_visible", store=True)
```

Comment:
**suggestion(non-blocking):** Make sure that this field is computing properly and working as expected

Reply:
@renda-dev  is this comment still relevant or can it already be closed?

Reply:
if the field is working as expected, I think it can be closed